cli_reference/osdk/cli-osdk-install.adoc

```diff
@@ -8,7 +8,7 @@ toc::[]
 
 The Operator SDK provides a command-line interface (CLI) tool that Operator developers can use to build, test, and deploy an Operator. You can install the Operator SDK CLI on your workstation so that you are prepared to start authoring your own Operators.
 
-Operator authors with cluster administrator access to a Kubernetes-based cluster, such as {product-title}, can use the Operator SDK CLI to develop their own Operators based on Go, Ansible, or Helm. link:https://kubebuilder.io/[Kubebuilder] is embedded into the Operator SDK as the scaffolding solution for Go-based Operators, which means existing Kubebuilder projects can be used as is with the Operator SDK and continue to work.
+Operator authors with cluster administrator access to a Kubernetes-based cluster, such as {product-title}, can use the Operator SDK CLI to develop their own Operators based on Go, Ansible, Java, or Helm. link:https://kubebuilder.io/[Kubebuilder] is embedded into the Operator SDK as the scaffolding solution for Go-based Operators, which means existing Kubebuilder projects can be used as is with the Operator SDK and continue to work.
```

Comment:
I don't see this addition of Java reflected in the preview:
![Screenshot 2023-04-19 at 10 06 16 AM](https://user-images.githubusercontent.com/14860856/233118968-dee9ade2-3977-4a8c-8512-002f85eb3651.png)
am I missing something?

Reply:
https://58866--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/osdk/cli-osdk-install.html ==> It's now showing here.

Reply:
Updated comment 0 with the preview link.